### PR TITLE
Refactors description func of NSDictionary

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

**Comment by spevans** [comment]
@swift-ci test

File: Sources/Foundation/NSDictionary.swift [hunk 1]
```diff
@@ -372,46 +372,47 @@ open class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCoding,
...
+            case let nsDecimalNumber as NSDecimalNumber:
+                line += nsDecimalNumber.description(withLocale: locale)
+            case let nsDictionary as NSDictionary:
+                line += nsDictionary.description(withLocale: locale, indent: level + 1)
+            case let nsOrderedSet as NSOrderedSet:
+                line += nsOrderedSet.description(withLocale: locale, indent: level + 1)
+            case let nsSet as NSSet:
+                line += nsSet.description(withLocale: locale)
+            case let hashableObject as Dictionary<AnyHashable, Any>:
+                line += hashableObject._nsObject.description(withLocale: nil, indent: level+1)
```
Comment (spevans) [inline Sources/Foundation/NSDictionary.swift]: Minor nit, fix the white space around `level+1`, thanks.

**Comment by spevans** [comment]
This looks fine apart from the tiny white space fix. If you could fix that up and squash all of the commits into one then this can be merged (Ignore the macOS test failure, it is unrelated).

File: Sources/Foundation/NSDictionary.swift (hunk 1 above)
Comment (valeriyvan) [reply Sources/Foundation/NSDictionary.swift]: fixed

**Comment by valeriyvan** [comment]
fixed spacing and squashed

**Comment by spevans** [comment]
@swift-ci test

**Comment by spevans** [comment]
@swift-ci test

**Comment by spevans** [comment]
@swift-ci test

**Comment by spevans** [comment]
@swift-ci test

**Pull request merged**